### PR TITLE
added gecko vendor for amp-embed

### PR DESCRIPTION
**Pull request by reemeng**

### CODE REVIEW

File: ads/vendors/gecko.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/gecko.js:2]: Nit: 2021 (and in all other files)

**Review by powerivq — COMMENTED** [review]
Also needs to create 3p/gecko.js (follow other files' pattern)

File: ads/vendors/gecko.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vendors/gecko.md:2]: Nit: 2021

**Review by powerivq — CHANGES_REQUESTED** [review]

File: ads/vendors/gecko.md (hunk 2 above)
Comment (reemeng) [reply ads/vendors/gecko.md]: Hello Shihua,
Thank you very much, can you please check now? I've fixed all the issues you raised

**Review by powerivq — APPROVED** [review]

**Comment by reemeng** [comment]
Hello,
Do you know how much time it will take till it will be merged?

Sincerely,
Reem

File: ads/vendors/gecko.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    publisherId: data['publisherid'],
+    url: data['url'] || global.context.canonicalUrl,
+    ampURL: data['ampurl'] || global.context.sourceUrl,
+    mode: data['mode'] || 1,
+    style: data['stylecss'] || '',
+    referrer: global.context.referrer,
+  };
+
+  loadScript(global, 'https://widget.gecko.me/gecko/gecko_amp_loader.js');
+
```
Comment (powerivq) [inline ads/vendors/gecko.js:39]: Nit: remove extra line

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
@reemeng Hello. Please fix this nit issue. It fails the lint check and that is why tests are not passing.

File: ads/vendors/gecko.js (hunk 3 above)
Comment (reemeng) [reply ads/vendors/gecko.js]: fixed :)

**Review by powerivq — APPROVED** [review]

**Comment by reemeng** [comment]
Hello,
Can you please let us know when it will be live?

Sincerley,
Reem

**Comment by powerivq** [comment]
If everything goes fine, it will be live mid next week.

**Comment by reemeng** [comment]
Hello,
Do you know why does it takes so much time?

Sincerely,
Reem

**Comment by powerivq** [comment]
@reemeng I checked and saw that `gecko` has been live in the stable version for a while. It should be working. What problem are you seeing now?

**Comment by reemeng** [comment]
Hello,
Thank you for your response, but we're getting this error:
Error: Embed type gecko not allowed with tag AMP-EMBED.
Can you please assist?

Sincerely,
Reem

**Comment by powerivq** [comment]
@reemeng That is expected because `gecko` is not allowlisted under amp-embed, so it can only be used in amp-ad.

**Pull request merged**